test/versioned/aws-sdk-v3/client-dynamodb.test.js

```diff
@@ -24,7 +24,14 @@ test('DynamoDB', async (t) => {
     })
 
     ctx.nr.server = server
-    ctx.nr.agent = helper.instrumentMockedAgent()
+    ctx.nr.agent = helper.instrumentMockedAgent({
```

Comment:
i would also add a test where you don't set account_id and assert we don't add `cloud.resource_id`

Reply:
Done